### PR TITLE
Adding support for credentials 3 and 4

### PR DESCRIPTION
**Pull request by AnotherJoSmith**
@andrewpaliga can you review this?

### CODE REVIEW

File: test/unit/integrations/universal/universal_helper_test.rb [hunk 1]
```diff
@@ -49,6 +55,13 @@ def test_core_fields
     assert_field 'x_test', @test.to_s
   end
 
+  def test_empty_credential_field_not_present_in_request
+    @options[:credential4] = ''
```
Comment (andrewpaliga) [inline test/unit/integrations/universal/universal_helper_test.rb]: I'd test both 3 and 4 here

File: test/unit/integrations/universal/universal_helper_test.rb [hunk 2]
```diff
@@ -7,6 +7,8 @@ def setup
     @order = 'order-500'
     @account = 'zork'
     @key = 'TO78ghHCfBQ6ZBw2Q2fJ3wRwGkWkUHVs'
+    @credential3 = '123456789'
+    @credential4 = 'email@gmail.com'
```
Comment (andrewpaliga) [inline test/unit/integrations/universal/universal_helper_test.rb]: I'd use something more generic than email here to avoid confusion. ex. abcdefgh

**Comment by andrewpaliga** [comment]
LGTM -- adding @girasquid

**Comment by girasquid** [comment]
:+1:

**Comment by AnotherJoSmith** [comment]
Should we bump the version of the gem after this?

**Pull request merged**